src/Hubbup.Web/Views/IssueList/_RepoList.cshtml

```diff
@@ -13,9 +13,10 @@
         </tr>
     </thead>
     <tbody>
+        @{var rowsRendered = 0; }
```

Comment:
Can wrap this `var` and the `if` in one big `@{ ... ... ... }`. Might be a little cleaner that way.